### PR TITLE
Update manifests: 86Box.86BoxManager version 1.7.4

### LINKED ISSUE #162018: [Update Request]: Correct package name for 86Box Manager
### What type of update are you requesting?

A change to the package metadata

### Current Package Identifier

86Box.86BoxManager

### Package Version

1.7.4

### Please describe the changes you would like to see

PackageName for the en-US locale is currently misspelled as "86BoxMamager". This should be corrected to "86Box Manager" (adding a space as well since that is how it is stylized in [the repository's current README](https://github.com/86Box/86BoxManager/blob/ce6e51bf1c363a6fdf4c935650d6ead59fa77230/README.md)).

### PR DESCRIPTION
**Pull request by Dragon1573**
## Description

- Change `PackageName` of the package.

## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- Resolve #162018

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/162058)

### CODE REVIEW

File: manifests/8/86Box/86BoxManager/1.7.4/86Box.86BoxManager.locale.en-US.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ Publisher: 86Box Project
 PublisherUrl: https://86box.net
 PublisherSupportUrl: https://paypal.me/86Box
```
Comment (Trenly) [inline manifests/8/86Box/86BoxManager/1.7.4/86Box.86BoxManager.locale.en-US.yaml:6]: I know it wasn't part of the initial change, but this doesn't look like a valid support URL

**Comment by Trenly** [comment]
[Policy] Needs author Feedback

File: manifests/8/86Box/86BoxManager/1.7.4/86Box.86BoxManager.locale.en-US.yaml (hunk 1 above)
Comment (stephengillie) [reply manifests/8/86Box/86BoxManager/1.7.4/86Box.86BoxManager.locale.en-US.yaml:6]: I've added `paypal` to my "word filter list", to help mitigate this in the future.

Comment (stephengillie) [reply manifests/8/86Box/86BoxManager/1.7.4/86Box.86BoxManager.locale.en-US.yaml:6]: Hi @Dragon1573,

Just commenting out the `PublisherSupportUrl` would probably be enough here.

Comment (Dragon1573) [reply manifests/8/86Box/86BoxManager/1.7.4/86Box.86BoxManager.locale.en-US.yaml:6]:
```suggestion
# PublisherSupportUrl:
```

**Review by Trenly — APPROVED** [review]

**Comment by stephengillie** [comment]
- https://github.com/microsoft/winget-pkgs/issues/142854

**Pull request merged**